<?xml version="1.0"?>
<!--This launches all the rustbot system-->
<launch>
<arg name="bag" default="zz_2017-10-10-13-08-55"/>
<arg name="local" default="/home/felipe/bagfiles/"/>

<!-- ROSBAG -->

  <!--Parameters-->
  <param name="/use_sim_time" value="true"/>
  <!--uncompress images-->
  <node pkg="image_transport" type="republish" name="republish_left" output="screen" args="compressed in:=/stereo/left/image_raw raw out:=/stereo/left/image_raw"/>
  <node pkg="image_transport" type="republish" name="republish_right" output="screen" args="compressed in:=/stereo/right/image_raw raw out:=/stereo/right/image_raw"/>
  <node pkg="image_transport" type="republish" name="republish_left_color" output="screen" args="compressed in:=/stereo/left/image_color raw out:=/stereo/left/image_color"/>
  <node pkg="image_transport" type="republish" name="republish_right_color" output="screen" args="compressed in:=/stereo/right/image_color raw out:=/stereo/right/image_color"/>
  <node pkg="rosbag" type="play" name="rosbag_play" output="screen" args=" $(arg local)/$(arg bag).bag --clock -r 0.22"/>
      
<!-- ALL.LAUNCH 
    <group ns="all">
      <include file="$(find rustbot_bringup)/launch/all.launch"> </include>
    </group> -->
      

</launch>
